fairlearn/metrics/_group_metric_result.py

```diff
@@ -69,7 +69,7 @@ def argmax_set(self):
         return set([k for k, v in self.by_group.items() if v == self.maximum])
 
     @property  # noqa: A003
-    def range(self):
+    def range(self):  # noqa: A003
```

Comment:
Thank you. In view of the discussions about this whole type, I think that's the best option :-)